Playbooks/MitreMapper/other/MitreMappingPlaybook.json

```diff
@@ -0,0 +1,907 @@
+                    "For_each": [
+                        "Succeeded"
+                    ]
+                },
+                "type": "ApiConnection"
+            },
+            "all_dtes": {
+                "inputs": {
+                    "variables": [
+                        {
+                            "name": "all_dtes",
+                            "type": "string"
+                        }
+                    ]
+                },
+                "runAfter": {},
+                "type": "InitializeVariable"
+            },
+            "get_technique_and_description": {
+                "inputs": {
+                    "body": "let attack=_GetWatchlist('attack');\nattack\n|where Technique==\"@{variables('Technique')}\"\n|project TechniqueName, TechniqueDescription\n|top 1 by TechniqueName",
+                    "host": {
+                        "connection": {
+                            "name": "@parameters('$connections')['azuremonitorlogs_1']['connectionId']"
+                        }
+                    },
+                    "method": "post",
+                    "path": "/queryData",
+                    "queries": {
+                        "resourcegroups": "CISOSentinelLogAnalytics_RG",
+                        "resourcename": "CISOLogAnalyticsWorkspace",
+                        "resourcetype": "Log Analytics Workspace",
+                        "subscriptions": "2b1ac3c6-a8f3-4c31-aed6-560d2ad57fd8",
+                        "timerange": "Last hour"
+                    }
+                },
+                "runAfter": {
+                    "Update_incident": [
+                        "Succeeded"
+                    ]
+                },
+                "type": "ApiConnection"
+            }
+        },
+        "contentVersion": "1.0.0.0",
+        "outputs": {},
+        "parameters": {
+            "$connections": {
+                "defaultValue": {},
+                "type": "Object"
+            }
+        },
+        "triggers": {
+            "When_Azure_Sentinel_incident_creation_rule_was_triggered": {
+                "inputs": {
+                    "body": {
+                        "callback_url": "@{listCallbackUrl()}"
+                    },
+                    "host": {
+                        "connection": {
+                            "name": "@parameters('$connections')['azuresentinel_2']['connectionId']"
+                        }
+                    },
+                    "path": "/incident-creation"
+                },
+                "type": "ApiConnectionWebhook"
+            }
+        }
+    },
+    "parameters": {
+        "$connections": {
+            "value": {
+                "azuremonitorlogs_1": {
+                    "connectionId": "/subscriptions/2b1ac3c6-a8f3-4c31-aed6-560d2ad57fd8/resourceGroups/CISOSentinelLogAnalytics_RG/providers/Microsoft.Web/connections/azuremonitorlogs-6",
+                    "connectionName": "azuremonitorlogs-6",
```

Comment:
@socinabox: ARM Template JSON contains organizational information such as subscription Id, Resource Group names, duplicate connection strings. Please use https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator to generate ARM Templates